src/routes/index.js

```diff
@@ -1 +1,59 @@
 // ADD CODE HERE
+var express = require('express');
+const bodyParser = require('body-parser')
+const { getAllTodos, getCompletedTodos,completeOneTodo,addOneTodo } = require('./routes/index.js')
+const app = express()
+app.set('view engine', 'pug')
+app.use(express.static('public'))
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true}))
+
```

Comment:
Lines 3-9 are not needed in this file.
The app.js file utilizes the `.use` method to link up the routes folder. 
The top lines of this file should just be
```
var express = require('express');
var app = express.router();
```